tools/hangar/__snapshots__/invalid.ts.snap

```diff
@@ -793,14 +793,14 @@ Duration <DURATION>"
 `;
 
 exports[`cloud_function_expects_inflight.test.w 1`] = `
-"error: Expected type to be \\"inflight (event: str): void\\", but got \\"preflight (name: str): str\\" instead
+"error: Expected phase to be preflight, but got inflight instead
```

Comment:
Looks like the error message got swapped around - it should say that the expected phase is "inflight"

Reply:
nice catch